common/src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -137,7 +137,7 @@ public static ECKeyPair fromSeed(byte[] seed) {
     }
 
     try {
-      return fromPrivateKey(HashUtils.blake2b256(seed).asBytes());
+      return fromPrivateKey(HashUtils.sha256Twice(seed).asBytes());
```

Comment:
I ack that this test does not rely that much on what hash function is used, but for consistency it would still be cool to use blake2b "everywhere"...?

**A)** I thought that the fix would simply be to call a `TestSetupUtils.installBouncyCastleProvider();` (as many other unit tests do). It seems to do the job on my machine.

**B)** Alternatively, we could stop relying on the "security provider" infra for our hashing purposes. I mean, in the `Blake2BHashHandler` we could simply do this instead:
```
  private final ThreadLocal<MessageDigest> blake2BDigester =
      ThreadLocal.withInitial(Blake2b.Blake2b256::new);
```

(then, many `Security.insertProviderAt(new BouncyCastleProvider(), 1);` lines could be dropped; although probably not all, because I see we want to rely on BouncyCastle for SecureRandom)

Please let me know if any of these options make sense here. If they are not worth the hassle, you can also feel free to merge as-is 👍

Reply:
Yup, totally agree, was just being lazy :) Will go with option **A**.